### DISCUSSION TITLE
eBPF support for Windows Server 2016?

### DISCUSSION THREAD

**Discussion by herms14** [microsoft/ebpf-for-windows#3915] [Q&A] [answered]
Hi Team,

I have a customer who wants to use eBPF for Windows Server 2016. Can anyone confirm that this is supported as the documentation states 2019 or later?

**Comment by herms14** [comment]
Hi Team,

any information on this?

**Comment by shankarseal** [comment, marked as answer]
Unfortunately, Windows server 2016 is not supported as it has reached [end of life](https://learn.microsoft.com/en-us/lifecycle/products/windows-server-2016).
